src/app/design/frontend/base/default/template/shopgate/cloudapi/header/checkout/onepage/success.phtml

```diff
@@ -76,7 +76,7 @@
   document.addEventListener('DOMContentLoaded', exchangeContinueShoppingButton);
 
   function exchangeContinueShoppingButton() {
-    if (<?php echo $this->isShopgateCheckout() ? 'false' : 'true'; ?>) {
+    if (<?php echo $this->isShopgateRequest() ? 'true' : 'false'; ?>) {
```

Comment:
the method isShopgateRequest is not available in the block "Shopgate_Cloudapi_Block_Checkout_Onepage_Success"

Reply:
File was missing in the branch, i added it